pkg/cover/backend/backend.go

```diff
@@ -9,11 +9,17 @@ import (
 	"github.com/google/syzkaller/sys/targets"
 )
 
+type KernelModule struct {
+	Name string `json:"name"`
+	Path string `json:"path"`
+	Addr uint64 `json:"addr"`
+}
+
 type Impl struct {
 	Units     []*CompileUnit
 	Symbols   []*Symbol
 	Frames    []Frame
-	Symbolize func(pcs []uint64) ([]Frame, error)
+	Symbolize func(pcs []uint64, modules []KernelModule) ([]Frame, error)
```

Comment:
We already pass modules to makeELF, I think the backend needs to remember modules passed to constructor and use these rather than requiring passing them to various callbacks again. In end, modules passed to Symbolize can be different from modules passed to Make, then it's quite unclear how to handle it.